helpers/api.py

```
@@ -0,0 +1,58 @@
+"""Pair Locust and Slumber to allow easier load testing of REST APIs.
+
+See: http://www.renzolucioni.com/pairing-locust-and-slumber/.
+"""
+from edx_rest_api_client import exceptions
+from edx_rest_api_client.client import EdxRestApiClient
+import requests
+import slumber
+
+
+class LocustResource(slumber.Resource):
+    """Custom Slumber Resource which takes advantage of Locust's extended HttpSession."""
+    def _request(self, method, data=None, files=None, params=None):
```

@jimabramson instead of silencing that kind of error, wouldn't we prefer this to fail loudly so we can investigate the change to Slumber and modify this accordingly?